src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -464,6 +464,7 @@ private static void MapPadding(IButtonHandler handler, Button button)
 		public static void MapText(IButtonHandler handler, Button button)
 		{
 			handler.PlatformView?.UpdateText(button);
+			button.FireMeasureChanged(Internals.InvalidationTrigger.MeasureChanged, 0);
```

Comment:
Like happens with Editor or Label, is not necessary to invalidate measure on other platforms too?

Reply:
Yea, I was trying to be minimal but actually I just noticed that we already have this on the button BPs

![image](https://github.com/user-attachments/assets/aa9d5c84-4c82-4fc9-9fe5-c1a986a49887)

So it was already firing via that. 

I deleted the code from the BP and moved this so all the mappers now just fire.

Good find!